tests/managers/block_tests.py

```diff
@@ -181,6 +196,27 @@ def test_cancel_snapshot_immediately(self):
             identifier=123,
         )
 
+    def test_cancel_snapshot_immediately_hourly_billing(self):
+        mock = self.set_mock('SoftLayer_Network_Storage', 'getObject')
+        mock.return_value = {
+            'billingItem': {
+                'activeChildren': [
+                    {'categoryCode': 'storage_snapshot_space', 'id': 417}
+                ],
+                'hourlyFlag': True,
+                'id': 449
+            },
+        }
+
+        self.block.cancel_snapshot_space(1234, immediate=True)
```

Comment:
Based on the current logic in the Block (and File) managers, immediate will always be used if the volume has hourly billing. Passing in False in the unit test is meant to ensure that the immediate flag is correctly changed to be True when the API call is actually made.